.solr_wrapper.yml

```diff
@@ -1,5 +1,6 @@
 # Place any default configuration for solr_wrapper here
 # port: 8983
+version: 8.11.2
```

Comment:
Was this change needed to get the tests to pass?  Does BAL not work with Solr 9?

Reply:
No, it just made it easier working on my local. I'll remove this change.